django/test/runner.py

```diff
@@ -437,7 +484,8 @@ def __init__(self, pattern=None, top_level=None, verbosity=1,
                  interactive=True, failfast=False, keepdb=False,
                  reverse=False, debug_mode=False, debug_sql=False, parallel=0,
                  tags=None, exclude_tags=None, test_name_patterns=None,
-                 pdb=False, buffer=False, enable_faulthandler=True, **kwargs):
+                 pdb=False, buffer=False, enable_faulthandler=True,
+                 timing=-1, **kwargs):
```

Comment:
Pytest uses `durations` rather than timing. Any benefit in aligning? 

https://docs.pytest.org/en/stable/usage.html#profiling-test-execution-duration

Reply:
I think there is, just so folks have a little less to learn between the two.

Reply:
It would be nice to align it, considering we've aligned similar pytest features.